pkg/kubecost/allocation.go

```diff
@@ -1527,12 +1625,6 @@ func (as *AllocationSet) accumulate(that *AllocationSet) (*AllocationSet, error)
 		return as, nil
 	}
 
-	if that.Start().Before(as.End()) {
```

Comment:
Good question. Short answer is: yes, that's right.

Context is that we originally wanted to make sure that the windows didn't overlap so that we could add `Minutes`. Turned out, adding minutes wasn't mathematically kosher, anyways. And if you want to accumulate two sets with overlapping windows, it should be up to you! We don't, so it shouldn't be an issue anyways.